src/dat/view.cljs

```diff
@@ -266,14 +266,14 @@
 (representation/register-representation
   ::pull-summary-string
   (fn [_ _ pull-data]
-    ;[:span
+    [:span
```

Comment:
Gotcha. Ok. I think there's something else though that needs a string directly. So for now let's make anything that needs a valid component use the `::pull-summary` component instead (if possible).